Ruby Basic/task4.rb

```diff
@@ -0,0 +1,21 @@
+puts 'input boolean true or false:'
+
+input_value = gets.chomp.downcase
+
+boolean_keywords = %w[true false]
+
+abort('You need enter true/false keyword.') unless boolean_keywords.include? input_value
+
+input_value = input_value == true
```

Comment:
When you send argument `true` in script, script will take it as string, not as a boolean. So, when you send `true`, in `input_value` will be `'true'` as string, and this line `input_value = input_value == true` will work incorrectly